### ISSUE TITLE
No update of task date fields possible

### ISSUE THREAD

**Issue by bachmi** [dmfs/opentasks#837]
Since if have upgraded my Pixel 3 smartphone to Android version 10 last Friday, I am not able to update date fields "Beginn" and "Fällig" of an existing task anymore. All other changes on e.g. description, priority and also app's own checklist are working fine.
If trying to change one or both of the date fields and submit with save, the app mostly returns to detailed (read-only) view of the specific task with no change. Sometimes the whole app crashes and Android popup window asks to close the app.

I am using DAVx5 to sync my CALDAV calendars.

OpenTasks Version: 1.1.20
Phone : Google Pixel 3
Android Version: 10

**Comment by dmfs** [comment]
Could be related to #823. I'll try this on Android 10.

**Comment by dmfs** [comment]
Does this also happen when updating the due date in the details view (using the +1h or +1d buttons)?

**Comment by bachmi** [comment]
No. When using the +1h or +1d buttons in the details view, due date is updated and stored correctly. This will help me to have a workaround. Thanks
